### PR TITLE
kube/ignition: Add service_disabled template definition

### PR DESCRIPTION
**Pull request by JAORMX**
This adds a template definition for the kubernetes and ignition
remediations. This results in having fixes generated automatically for
disabled services.

### CODE REVIEW

**Comment by matejak** [comment]
test this please

File: ssg/templates.py [hunk 1]
```diff
@@ -286,7 +286,7 @@ def sebool(data, lang):
     return data
 
 
-@template(["ansible", "bash", "oval", "puppet"])
+@template(["ansible", "bash", "oval", "puppet", "ignition", "kubernetes"])
```
Comment (jan-cerny) [inline ssg/templates.py:289]: This needs to be added to the documentation in docs/manual/developer_guide.adoc line 1622.

File: shared/templates/template_IGNITION_service_disabled [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    ignition:
+      version: 2.2.0
+    systemd:
+      units:
+      - name: {{{ DAEMONNAME }}}.service
+        enabled: false 
+{{%- if MASK_SERVICE %}}
+        mask: true
+{{%- endif %}}
+      - name: {{{ DAEMONNAME }}}.socket
```
Comment (jan-cerny) [inline shared/templates/template_IGNITION_service_disabled:19]: What will happen if the socket file doesn't exist?

File: shared/templates/template_IGNITION_service_disabled [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ocp
+# reboot = true
```
Comment (jan-cerny) [inline shared/templates/template_IGNITION_service_disabled:2]: Is a reboot needed?

Comment (JAORMX) [reply shared/templates/template_IGNITION_service_disabled:2]: Yes. It happens whenever you add/modify a MachineConfig. The machine-config-operator does that.

File: shared/templates/template_IGNITION_service_disabled (hunk 2 above)
Comment (JAORMX) [reply shared/templates/template_IGNITION_service_disabled:19]: AFAIU, it shouldn't be a problem as it should just remove the file in the targets directory. Or if "masked" is enabled, it should just point it to /dev/null

File: ssg/templates.py (hunk 1 above)
Comment (JAORMX) [reply ssg/templates.py:289]: Done

**Comment by jan-cerny** [comment]
Thank you very much!

**Pull request merged**